### ISSUE TITLE
UTF-8-BOM files generate ORA-00911 Error

### ISSUE THREAD

**Issue by PsymonK** [dbeaver/dbeaver#4164]
Loading existing SQL-Files saved as UTF-8-BOM do generate an ORA-00911 Error on my Win7 Pro Machine.
Converting the file to UTF-8 (without BOM) with Notepad++ and afterwards loading in the SQL Editor is working, but a real pain in the *#%.
I've just switched to DBeaver from another Software and would like to stay, cause I really like it. But finding this error has nearly cost my 1 working day...

Please fix this one!

Regards

**Comment by serge-rider** [comment]
Generally SQL files with BOM do not cause any problems. Because DBeaver recognizes BOM. You can check this by pressing alt+enter in open script editor:
![image](https://user-images.githubusercontent.com/6398845/45175085-364eee80-b215-11e8-98f0-6d54b8ac2273.png)
Is it possible that you have some other non valid  UTF-8 characters in the script?

**Comment by serge-rider** [comment]
Another question - do you use DBeaver SQL scripts or you link some external folder/file?

**Comment by PsymonK** [comment]
I'm opening a file from my harddrive in the SQL Editor.

There are definitly no non valid UTF characters in the file.
I've opened my file, have run the script via Alt+x, got ORA-00911 error.
Then I'd copied the script from this SQL Editor Windows to an new one, step by step. Cause I thought there must be a failure in my script (although it was running in another SQL Manager).
Next step was saving the script to my harddrive. I compared the 2 files with Notepad++ and the only difference was that my file is UTF-8-BOM and the other is UTF-8.
There is no execution error when loading the UTF-8 file again.

Will check the file properties in the SQL Editor tomorrow.
